pages/docs/installation/installation-source-discoverability.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Building from source - Discoverability
+permalink: installation-source-discoverability.html
+keywords: configuration, basics, cmake, installation, building, source, bash, profile
+toc: false
+---
+
+If you installed preCICE in a custom prefix, then one still needs to make it discoverable by the system.
+
+The preCICE library needs to be discoverable in various ways:
+
+1. The location of headers needs to be available during compilation.
+   Compilers use hints from `CPATH` if no extra compilation flags were passed to them using `-I`.
+2. the shared library needs to be available during linking and execution.
+   The dynamic linker uses hints from `LD_LIBRARY_PATH` (`DYLD_LIBRARY_PATH` on MacOS X).
+3. an optional step is to make the preCICE executables available using `PATH`.
+
+As this setup can become tedious to maintain, there are some tools that expose the above details.
+However, they also need to find preCICE in some way:
+
+* `pkg-config` and `pkgconfig` use `PKG_CONFIG_PATH` to search for additional `.pc` files.
+* `CMake` uses `CMAKE_PREFIX_PATH` for additional installation prefixes.
+  Alternatively one can specify the location of the preCICE configuration file using `precice_DIR`.
+
+## Using the shell
+
+If you are using a Unix-like system, you are using a shell, which is an easy and straight forward way of making preCICE discoverable.
+
+Let `PRECICE_PREFIX` be the chosen installation prefix.
+Then add the following to your `.profile` (for bash) or `.zshrc` (for zsh).
+
+```bash
+PRECICE_PREFIX=~/software/prefix # set this to your selected prefix
+export PATH=$PRECICE_PREFIX/bin:$PATH
```

Comment:
I know that this was already like this before, but try `shellcheck` on this. I would at least add double quotes to treat any spaces in the paths.

Reply:
I used shellcheck. No issues detected.

I expect that spaces aren't an issue as the paths are colon-separated.

Reply:
Interestingly, this depends on the shell. With `/bin/sh`, shellcheck 0.8.0 complains about the missing double quotes. With `/bin/bash`, it does not.

Since in other places in the documentation (e.g., in [Dependencies](https://precice.org/installation-source-dependencies.html) and in [Installation](https://precice.org/installation-source-installation.html)) we use the variant without quotes, let's stick with that for consistency.